lib/heartcheck/resque/version.rb

```diff
@@ -1,5 +1,5 @@
 module Heartcheck
   module Resque
-    VERSION = '0.0.2'
+    VERSION = '0.0.3'.freeze
```

Comment:
I think it is not necessary to generate a new version of the gem, just because something was fixed on README.

The code that is used by the applications stills the same, isn't it?

WDYT?

Reply:
I agree. We will keep the same version.

Thanks!